docs/guides/sqlite3_db_with_concord.md

```diff
@@ -0,0 +1,259 @@
+# Other topics
+
+## Using a DB with concord
+
+Sometimes, when you are developing a command, you may wish to save some data, that will be not deleted across the restarts from your bot, this is where a database comes in. It can save data, and, don't worry, it will not be deleted if you restart your bot.
+
+In this guide, we will be teaching you how to use sqlite3, a local database, that it's incredibly fast, easy to use and debug.
+
+### Opening the DB file
+
+The first step in this guide will be how to open the DB file, as it's required for the other sqlite3 functions like read and write data.
+
+```c
+#include <sqlite3.h>
+
+...
+
+sqlite3 *db;
+int rc = sqlite3_open("db.sqlite", &db);
+
+if (rc != SQLITE_OK) {
+    log_fatal("[SQLITE] Error when opening the db file. [%s]", sqlite3_errmsg(db));
+      
+    if (sqlite3_close(db) != SQLITE_OK) {
+        log_fatal("[SQLITE] Failed to close sqlite db. [%s]", sqlite3_errmsg(db));
+        abort();
+    }
+    return;
+}
+```
+
+In the first line, we are simply including the sqlite3 into the file.
```

Comment:
"In the first line, we are simply including the sqlite3 header into the file we are compiling."

Reply:
:+1:  :)